src/js/modal/login-wallet/modal.js

```diff
@@ -85,7 +85,12 @@ export default Modal.extend({
 
 		this.$("#wallet-password").on("keyup change click", () => {
 			let password = this.$("#wallet-password").val();
-			let strength = zxcvbn(password);
+			if (password.length >= 128) {
+				let score = 5;
```

Comment:
The `score` variable needs to be moved to an outer scope, it is only accessible within the if statement here. [Read about let here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let).

Reply:
Lignum's review is better